src/main/java/uk/gov/hmcts/reform/hmc/constants/Constants.java

```diff
@@ -24,7 +24,7 @@ private Constants() {
     public static final String TOPIC_HMC_TO_CFT = "hmc-to-cft";
 
     public static final String ERROR_PROCESSING_MESSAGE = "Error occurred during service bus processing. "
-        + "Service:{} . Entity: {}. Method: {}. Hearing ID: {}.";
+        + "Service:{}. Entity:{}. Method:{}. Hearing ID: {}.";
```

Comment:
Why leave a space for the hearing Id?

Reply:
Dyna trace use the Log message pattern from [here](https://tools.hmcts.net/confluence/display/HMAN/HMC+Monitoring+Design)
The log message contains a space for the hearing id. Hence space is not removed